### PR TITLE
Update get_file()

### PR DESCRIPTION
**Pull request by mykhaly**
This change is [Reviewable](https://reviewable.io/reviews/openprocurement/openprocurement.client.python/32)

### CODE REVIEW

File: openprocurement_client/client.py [hunk 1]
```diff
@@ -285,8 +285,6 @@ def get_file(self, tender, url, access_token):
         parsed_url = urlparse(url)
```
Comment (ktarasz) [inline openprocurement_client/client.py:285]: Можливо зробити з нього keyword argumrenrt access_token=None, access_token треба передавати в любому випадку

**Comment by mykhaly** [comment]
Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion, some commit checks broke.

---

_[openprocurement_client/client.py, line 285 [r2]](https://reviewable.io:443/reviews/openprocurement/openprocurement.client.python/32#-KJFvTgDnpSbWllQt1rZ-r2-285:-KJGBd_oon2mmpJJvhfu:-998800834) ([raw file](https://github.com/openprocurement/openprocurement.client.python/blob/aabb39e4aa3c804f2c281fb357ea64e93f8d4c3a/openprocurement_client/client.py#L285)):_
<details><summary>

<i>Previously, ktarasz (Taras Kozlovskyi) wrote…</i></summary>



> Можливо зробити з нього keyword argumrenrt access_token=None, access_token треба передавати в любому випадку
> </details>
> Якщо access token треба передавати в любому випадку, то навіщо його робити ківордом?

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/openprocurement/openprocurement.client.python/32)_

**Comment by ktarasz** [comment]
Review status: 0 of 2 files reviewed at latest revision, 2 unresolved discussions, some commit checks broke.

---

_[openprocurement_client/client.py, line 284 [r3]](https://reviewable.io:443/reviews/openprocurement/openprocurement.client.python/32#-KJGDVWWKTmKoOJFG1A1:-KJGDVWWKTmKoOJFG1A2:302160420) ([raw file](https://github.com/openprocurement/openprocurement.client.python/blob/e2b8af3efd0b5aa24b98405d95945e58ad7ce522/openprocurement_client/client.py#L284)):_

> ``` Python
>         return self._get_tender_resource_item(tender, lot_id, "lots")
> 
>     def get_file(self, tender, url, access_token):
> ```

`def get_file(self, tender, url, access_token=None):`

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/openprocurement/openprocurement.client.python/32)_

**Comment by mykhaly** [comment]
Review status: 0 of 2 files reviewed at latest revision, 2 unresolved discussions.

---

_[openprocurement_client/client.py, line 284 [r3]](https://reviewable.io:443/reviews/openprocurement/openprocurement.client.python/32#-KJGDVWWKTmKoOJFG1A1:-KJGFaE0qtG2iLHGHRfG:-499146009) ([raw file](https://github.com/openprocurement/openprocurement.client.python/blob/e2b8af3efd0b5aa24b98405d95945e58ad7ce522/openprocurement_client/client.py#L284)):_
<details><summary>

<i>Previously, ktarasz (Taras Kozlovskyi) wrote…</i></summary>



> `def get_file(self, tender, url, access_token=None):`
> </details>
> Done.

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/openprocurement/openprocurement.client.python/32)_

**Pull request merged**